build-image/openhabian.conf

```diff
@@ -27,4 +29,4 @@ wifi_country=""
 
 # Java architecture mode
 # Use "32-bit" or "64-bit"
-java_arch="64-bit"
\ No newline at end of file
+java_arch="32-bit"
```

Comment:
change not useful here. 
did you notice that we have 3 files in build-image? 
openhabian.conf: -> local_test
openhabian.pi-raspbian.conf: -> build image for pi
openhabian.pine64-xenial.conf: -> build image for pine
...plus openhabian.conf.dist....

I think we need to cleanup this mess later....

Reply:
agree, I removed the change here it slipped in when working on the java stuff in parallel